### ISSUE TITLE
Unable to handle site combo boxes

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8830]
### Steps to reproduce:
open the page:
http://recordtv.r7.com/trabalhe-conosco
press m to go to the frame.
There is a small form to fill out.
Walk with arrows down until the nvda speak:
estado*
state*
now there's a combo box
In nvda the option to automatically enter navigation mode is disabled
Press enter to enter the box
Try to navigate with the arrows above or below the nvda does not announce anything.
A person with vision said that the problem is that the combo box is not open and is not showing the states.
If we do not press enter in the combo box and press alt + arrow below the focus of nvda goes to a text box where we can type the text to look for in the case.
The problem is that in this case I could even type the name of the state to search, however I have already seen several other boxes just like this that I have to type the text to search, however I do not know what to type to search, and in Portuguese alsothe problem of accentuation of the letters in which to enter in the search a word with accent and she in the box it is without accent the search of the box does not find the term searched.
Other problems, a person who tells you that on the screen the combo box is now open and the list of states is being displayed.
However we who use nvda can not see this list we are focused inside a text box.
If I press ESC to exit the text box and navigate with arrows to look up the list of states, there is another problem as soon as the focus arrives in the combo box the nvda announces merged combo box and the focus of nvda is automatically placed in the text box again.
### Actual behavior:
Pressing enter will not open the combo box.
When pressing alt + down arrow the combo box is opened however the focus of nvda goes to the text box.
The list with states is displayed on the screen but nvda does not gain focus in the list of states nor a form.
If you press esc to exit the text box and navigate to the combo box with the states, the focus is automatically placed in the text box again.
### Expected behavior:
This problem is not only on this page, it may be due to poor programming, however I found this same behavior on another page to send a resume.
Unfortunately this other page is private and I can not send it for your reviews.
It should be noted that the behavior seen on this page may be due to poor programming, more common programming.
#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 16149
#### Windows version:
10 17763.1
#### Name and version of other software in use when reproducing the issue:
chrome 69
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
yes
